modules/dbfs/multi.go

```diff
@@ -0,0 +1,226 @@
+package dbfs
+
+import (
+	"strconv"
+
+	"fmt"
+
+	"github.com/CodeCollaborate/Server/modules/patching"
+	"github.com/CodeCollaborate/Server/utils"
+)
+
+// TODO (wongb): Move this to config
```

Comment:
Honestly, I would be against moving these values to config at all. These values should be something that's found through trial and error and just kind of set, rather than be something that's tweaked by the end user. It just seems unnecessary to expose